### PR TITLE
Many SNEKs: schain, digitrust, pixel beacons, 3.0 compliance

### PR DESCRIPTION
**Pull request by aprakash-sovrn**
## Type of change

- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Added schain, digitrust, and pixel sync'ing to the sovrn adapter
Upgraded adapter to be 3.0 compliant

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Review by jrosendahl — COMMENTED** [review]
tests?

**Review by jrosendahl — APPROVED** [review]
looks good to me.

File: modules/sovrnBidAdapter.js [hunk 1]
```diff
@@ -27,7 +27,22 @@ export const spec = {
...
+          const bidRequestDigitrust = utils.deepAccess(bid, 'userId.digitrustid.data');
+          if (bidRequestDigitrust && (!bidRequestDigitrust.privacy || !bidRequestDigitrust.privacy.optout)) {
+            digitrust = {
+              id: bidRequestDigitrust.id,
+              keyv: bidRequestDigitrust.keyv
+            }
+          }
+        }
+        if (bid.schain) {
+          schain = schain || bid.schain;
```
Comment (weswhitney) [inline modules/sovrnBidAdapter.js:41]: why `schain = schain || bid.schain;`? just curious

**Review by weswhitney — APPROVED** [review]
just a comment for learning discussion

**Pull request closed without merging**